health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_cli.py

```
@@ -0,0 +1,92 @@
+    config_file, mapping_file = _write_config(tmp_path)
+    args = [
+        "--dcgm-addr",
+        "localhost:5555",
+        "--dcgm-error-mapping-config-file",
+        str(mapping_file),
+        "--config-file",
+        str(config_file),
+        "--port",
+        "2112",
+        "--state-file",
+        "/tmp/statefile",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/statefile"

(S108)